kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Permissions.kt

```diff
@@ -57,6 +57,8 @@ public fun Permission.toTranslationKey(): String? = when (this) {
 
     Permission.UseExternalStickers -> "permission.useExternalStickers"
     Permission.UseEmbeddedActivities -> "permission.useEmbeddedActivities"
+    Permission.ViewCreatorMonetizationAnalytics -> TODO()
+    Permission.UseSoundboard -> TODO()
```

Comment:
it's a draft pr because i don't know what belongs here

Reply:
Looks to just be the translation key, i would assume just
```suggestion
    Permission.ViewCreatorMonetizationAnalytics -> "permission.viewCreatorMonetizationAnalytics"
    Permission.UseSoundboard -> permission.useSoundboard
```
then add to the translation files those lines. Just add it to the root strings.properties i believe

Reply:
That one and the UK one yeah

Reply:
done